addons/l10n_syscohada/i18n/fr.po

```diff
@@ -0,0 +1,7802 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* l10n_syscohada
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 16.1alpha1+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-11-03 09:45+0000\n"
+"PO-Revision-Date: 2022-11-03 09:45+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_151
+#: model:account.account,name:l10n_syscohada.2_pcg_151
+#: model:account.account.template,name:l10n_syscohada.pcg_151
+msgid "ACCELERATED DEPRECIATION"
+msgstr "AMORTISSEMENT ACCÉLÉRÉ"
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_97
+#: model:account.account,name:l10n_syscohada.2_pcg_97
+#: model:account.account.template,name:l10n_syscohada.pcg_97
+msgid "ACCOUNTING DIFFERENCE ACCOUNTS"
+msgstr "COMPTES D'ÉCARTS COMPTABLES"
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_582
+#: model:account.account,name:l10n_syscohada.2_pcg_582
+#: model:account.account.template,name:l10n_syscohada.pcg_582
+msgid "ACCREDITIVES"
+msgstr "ACCRÉDITIFS"
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_183
+#: model:account.account,name:l10n_syscohada.2_pcg_183
+#: model:account.account.template,name:l10n_syscohada.pcg_183
+msgid "ACCRUED INTEREST ON DEBTS RELATED TO SHAREHOLDINGS"
+msgstr "INTÉRÊTS COURUS SUR DETTES RATTACHÉES À DES PARTICIPATIONS"
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_133
+#: model:account.account,name:l10n_syscohada.2_pcg_133
+#: model:account.account.template,name:l10n_syscohada.pcg_133
+msgid "ADDED VALUE (A.V.)"
+msgstr "VALEUR AJOUTEE (V.A.)"
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_251
+#: model:account.account,name:l10n_syscohada.2_pcg_251
+#: model:account.account.template,name:l10n_syscohada.pcg_251
+msgid "ADVANCES AND DEPOSITS PAID ON INTANGIBLE ASSETS"
+msgstr "AVANCES ET ACOMPTES VERSÉS SUR IMMOBILISATIONS INCORPORELLES"
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_252
+#: model:account.account,name:l10n_syscohada.2_pcg_252
+#: model:account.account.template,name:l10n_syscohada.pcg_252
+msgid "ADVANCES AND DEPOSITS PAID ON PROPERTY, PLANT AND EQUIPMENT"
+msgstr "AVANCES ET ACOMPTES VERSÉS SUR IMMOBILISATIONS CORPORELLES"
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_164
+#: model:account.account,name:l10n_syscohada.2_pcg_164
+#: model:account.account.template,name:l10n_syscohada.pcg_164
+msgid "ADVANCES RECEIVED AND BLOCKED CURRENT ACCOUNTS"
+msgstr "AVANCES REÇUES ET COMPTES COURANTS BLOQUÉS"
```

Comment:
@malb-odoo It does not need to be done now, but would it be an idea to remove the capital letters?

Reply:
Done